app/controllers/errors_controller.rb

```diff
@@ -0,0 +1,12 @@
+# Created app/controllers/errors_controller.rb for the not found (404)
+# and internal server error (500) pages.
+# Send an HTTP status of 404 or 500 according to error page being rendered.
+class ErrorsController < ApplicationController
+  def not_found
+    render(:status => 404)
```

Comment:
use the preferred style for hashes: `status: 404` so this should be `render status: 404`